.github/workflows/e2e-tests.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '~1.17'
+      - run: mkdir -p artifacts
```

Comment:
Had to create the artifacts dir. Calling ginkgo (v2) with --junit-report ./artifacts/junit-report.xml fails with file not found

Reply:
I don't think this is blocking but I wonder if we should be handling any stat or directory creation process in the e2e suite?

Reply:
I dived into this some more - it looks like ginkgo v2 just fires off an os.Create call so if the artifacts directory doesn't already exist, it will produce an error so creating it beforehand is fine with me for now.